### PR TITLE
Fix the minor issue in style module

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
### Description of the feature or fix

see the commit message for detailed info

### Checkpoints
- [X] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update CHANGELOG.md
- [ ] Update the documentation

### CODE REVIEW

File: src/lv_core/lv_style.c [hunk 1]
```diff
@@ -530,9 +530,8 @@ void _lv_style_set_ptr(lv_style_t * style, lv_style_property_t prop, const void
  *         Higher number is means better fit
  *         -1 if the not found (`res` will be undefined)
  */
-int16_t _lv_style_get_int(const lv_style_t * style, lv_style_property_t prop, void * v_res)
+int16_t _lv_style_get_int(const lv_style_t * style, lv_style_property_t prop, lv_style_int_t * res)
```
Comment (embeddedt) [inline src/lv_core/lv_style.c:520]: @kisvegabor I *think* there was a reason for keeping them as `void *`. Something to do with compiler warnings on certain platforms?

Comment (xiaoxiang781216) [reply src/lv_core/lv_style.c:520]: It's very dangerous to use void * here. For example:
```
bool clip_corner;
_lv_style_get_int(&style, LV_STYLE_CLIP_CORNER, &clip_corner);
```
The above code can be compiled without any warning, but buffer overwrite may happen on some platform if bool is smaller than lv_style_int_t.

Comment (xiaoxiang781216) [reply src/lv_core/lv_style.c:520]: @embeddedt what do your opinion?

Comment (kisvegabor) [reply src/lv_core/lv_style.c:520]: I also don't remember why we have used `void*`. We should add comments next time :slightly_smiling_face: 

I agree to use `lv_style_int_t` and if someone reports an issue we can improve it.

Comment (embeddedt) [reply src/lv_core/lv_style.c:520]: Sounds good.

Comment (embeddedt) [reply src/lv_core/lv_style.c:520]: @kisvegabor Is this ready to merge?

Comment (kisvegabor) [reply src/lv_core/lv_style.c:520]: From my side yes. Merging.

**Pull request merged**